packages/react-data-grid/src/ReactDataGrid.js

```diff
@@ -66,6 +66,8 @@ const ReactDataGrid = React.createClass({
     onCellsDragged: React.PropTypes.func,
     onAddFilter: React.PropTypes.func,
     onGridSort: React.PropTypes.func,
+    sortColumn: React.PropTypes.string,
+    sortDirection: React.PropTypes.oneOf(['ASC', 'DESC', 'NONE']),
```

Comment:
@jpdriver now prop type check is using `DEFINE_SORT` constants from `SortableHeaderCell.js` file. I think that moving these definition to other file needs separate PR, maybe global refactor of dealing with constant values as these.